examples/logger.py

```diff
@@ -3,13 +3,33 @@
 """
 
 import time
+import logging
 from typing import Any, Optional, Tuple
 
 from PyQt5.QtCore import QObject, pyqtSlot, pyqtSignal
 from PyQt5.QtWidgets import QWidget, QVBoxLayout, QPushButton, QTextEdit, QLabel, QDialogButtonBox
 
 from qiwis import BaseApp
 
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+class LoggingHandler(QObject, logging.Handler):
+    """
+    Handler for logger, handle input log by emit
+    """
+    signal = pyqtSignal(str)
+    def __init__(self, slotfunc, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
From this code, the reader is confused whether it calls the `__init__` of `QObject` or `logging.Handler`. If you inherit only one class, then the problem is resolved.